### PR TITLE
Revert "Revert "Notifications""

### PR DESCRIPTION
**Pull request by GyuoMi**
I'm stupid qwq

### CODE REVIEW

File: appointment-app/src/routes/app/+layout.server.js [hunk 1]
```diff
@@ -2,21 +2,33 @@ import database from '../api/database.js';
 import {_clientID} from '../login/+page.server.js';
 
 /** @type {import('./$types').LayoutServerLoad} */
-export async function load({cookies}) {
```
Comment (SahilDinanath) [inline appointment-app/src/routes/app/+layout.server.js:5]: here

File: appointment-app/src/routes/app/+layout.server.js [hunk 2]
```diff
@@ -2,21 +2,33 @@ import database from '../api/database.js';
...
 //if client has not logged in yet return an empty json
   let clientId = cookies.get('clientId');
   if (_clientID !== undefined) {
-    bookingDetailsFromDatbaseJson = await database.getJsonFromSelectQuery(
-      `select * from BOOKING_TBL where CLIENT_ID = ${clientId} AND EVENT_START > NOW()`
+
+    bookingDetailsFromDatabaseJson = await database.getJsonFromSelectQuery(
+      `select B.*, C.CLIENT_FIRST_NAME, C.CLIENT_LAST_NAME from BOOKING_TBL B 
+      JOIN(SELECT E.EVENT_ID, E.CLIENT_ID, C.CLIENT_FIRST_NAME, C.CLIENT_LAST_NAME FROM EVENT_TBL E JOIN CLIENT_TBL C 
+      ON E.CLIENT_ID = C.CLIENT_ID) AS C ON B.EVENT_ID = C.EVENT_ID where B.CLIENT_ID = ${_clientID} AND B.EVENT_START > NOW()`
```
Comment (SahilDinanath) [inline appointment-app/src/routes/app/+layout.server.js]: change ${_clientID} -----> clientId

File: appointment-app/src/routes/app/+layout.server.js [hunk 3]
```diff
@@ -2,21 +2,33 @@ import database from '../api/database.js';
...
-export async function load({cookies}) {
-  let bookingDetailsFromDatbaseJson;
+export async function load() {
+  let bookingDetailsFromDatabaseJson;
+  let allBookings;
+
 //checks if clientID is defined before trying to access database with this value
 //if client has not logged in yet return an empty json
   let clientId = cookies.get('clientId');
   if (_clientID !== undefined) {
```
Comment (SahilDinanath) [inline appointment-app/src/routes/app/+layout.server.js]: also here change to clientId

Comment (GyuoMi) [reply appointment-app/src/routes/app/+layout.server.js]: changed to clientId now

**Pull request merged**